cluster-scope/base/apps/daemonsets/kvm-device-plugin/machineconfigpool.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
```

Comment:
Same as above, please create new folder in [`cluster-scope/base/machineconfiguration.openshift.io`](https://github.com/operate-first/apps/tree/master/cluster-scope/base/machineconfiguration.openshift.io) named `machineconfigpools/mcp-kvm-device-plugin` and place it there. Then either pull those resources individually into a `cluster-scope/overlays/<env>/<cluster>` (or create a bundle in `cluster-scope/bundles`).

Please read https://github.com/operate-first/apps/blob/master/cluster-scope/README.md (or 
https://github.com/operate-first/blueprint/blob/main/adr/0009-cluster-resources.md + https://github.com/operate-first/blueprint/blob/main/adr/0015-cluster-resources-amendment.md )

Reply:
Done as well